pyomo/solvers/tests/checks/test_xpress_persistent.py

```diff
@@ -291,6 +291,9 @@ def test_nonconvexqp_locally_optimal(self):
 
         opt = pe.SolverFactory('xpress_direct')
         opt.options['XSLP_SOLVER'] = 0
+        # xpress 9.5.0 now defaults to trying (and failing) to solve this problem
+        # using the global solver. This option forces the use of the local solver.
+        opt.options['NLPSOLVER'] = 1
```

Comment:
This is most likely due to a change in the community license. The community license is the default license you get when you install Xpress via pip or conda and I suppose this is the license you are using for tests.
Pre 9.5 this license did not allow running the Xpress global solver. So Xpress did only aim for finding local optima here.
Starting with 9.5, the community license now allows using the global solver as well (with limits on the problem size). That is why Xpress now uses the global solver by default.